Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp

```diff
@@ -174,12 +174,14 @@ static ErrorOr<Optional<String>> generate_enum_initializer_for(StringView proper
         { "text_alignment"sv, "Gfx::TextAlignment"sv },
         { "focus_policy"sv, "GUI::FocusPolicy"sv },
         { "foreground_role"sv, "Gfx::ColorRole"sv },
+        { "background_role"sv, "Gfx::ColorRole"sv },
```

Comment:
This appears to have slipped in from the other PR.

Reply:
Oh yeah, I guess I'll change it when the other PR is accepted
Next time when I need to port multiple apps to the GML compiler I'll do it in a single PR